browser/css/device-tablet.css

```diff
@@ -43,7 +43,6 @@
 	align-items: center;
 }
 .hasnotebookbar {
-	overflow: auto hidden;
 	scrollbar-width: none;
 	-ms-overflow-style: none;
```

Comment:
This was added in the past to make sure we wouldn't have vertical scrolling because at the time the container was shorter than the tab children for some reason. This seems safe to remove particularly if the selection handles get hidden because of that line. Thanks @Minion3665 always good to remove stuff :)